packages/web-components/src/components/rux-card/rux-card.scss

```diff
@@ -30,11 +30,19 @@
     letter-spacing: var(--font-heading-3-letter-spacing);
     line-height: var(--font-heading-3-line-height);
     box-sizing: border-box;
+    text-align: left; // Solves the issue of a parent element with a different text-align overriding.
 }
 
 .rux-card__body {
     padding: var(--body-padding);
     box-sizing: border-box;
+    font-size: var(
+        --font-size-base
```

Comment:
lets swap the --base tokens with the typography tokens for body-1:

```
    font-family: var(--font-body-1-font-family);
    font-size: var(--font-body-1-font-size);
    font-weight: var(--font-body-1-font-weight);
    line-height: var(--font-body-1-line-height);
    letter-spacing: var(--font-body-1-letter-spacing);
```

Reply:
actually this applies to just about anything that uses --font-size-base as well

Reply:
This has been applied to all components where applicable, but I kept line-height as `--line-height-base` as this caused weird icon shift in `rux-global-status-bar` when I tried to apply the line-height for font-body-1.